be/src/storage/schema_change.cpp

```diff
@@ -642,6 +653,7 @@ bool ChunkSorter::sort(ChunkPtr& chunk, TabletSharedPtr new_tablet) {
     _swap_chunk = chunk->clone_empty_with_schema();
     _swap_chunk->append_selective(*chunk, selective.data(), 0, chunk->num_rows());
 
+    chunk->swap_chunk(*_swap_chunk);
```

Comment:
This is another bug introduced by this pr https://github.com/StarRocks/starrocks/pull/4799/files

Reply:
Split PR